### ISSUE TITLE
Roadmap to Pure ODD

### ISSUE THREAD

**Issue by musicEnfanthen** [music-encoding/music-encoding#1180]
-- Please feel free to add existing issues or plain text descriptions of needed steps --

Steps proposed in preparation to move to Pure ODD:

- [x] #304 
- [x] #991 
- [x] #1164 
- [x] #1375
- [ ] #1501
- [x] #1582
- [x] #1583

**Comment by pe-ro** [comment]
I don't think Pure ODD supports the concept of interleave.  This is a stumbling block to adopting Pure ODD in my opinion.  Perhaps Pure ODD supports interleave in a way that I'm not familiar with, though.

**Comment by ahankinson** [comment]
@pe-ro could you point to an example of where "interleave" is used in MEI?

**Comment by pe-ro** [comment]
It's used in the content model of `<ossia>` and in the modifications to `<work>`, `<expression>`, `<manifestation>`, and `<item>` I recently suggested.

I suppose the reason it's not supported in ODD is to allow for outputting XSD schemas, which also don't support it.  We decided long ago not to worry about providing an XSD for MEI.

**Comment by lb42** [comment]
I believe support for tei:interleave is now scheduled for the next tei release

**Comment by lb42** [comment]
It would be nice to see the other parts of this roadmap acted upon soon for the benefit of thise who dont actually use interleave!

**Comment by raffazizzi** [comment]
Yes @pe-ro we are working on it https://github.com/TEIC/TEI/pull/2538

**Comment by bwbohl** [comment]
Sitting together yesterday at the MEI Developer Workshop we cam up with the following thoughts:

- [ ] Get / find / implement a script to update automatically
- [ ] Make a clear announcement and declare a date for the switch to pureODD in order to not break pending pull requests and developments
- [ ] implement a GithubAction to transform dev to dev-pureODD automatically
